connect/runtime/src/test/java/org/apache/kafka/connect/runtime/errors/ErrorReporterTest.java

```diff
@@ -109,12 +109,10 @@ public void testDLQConfigWithEmptyTopicName() {
 
         ProcessingContext context = processingContext();
 
-        EasyMock.expect(producer.send(EasyMock.anyObject(), EasyMock.anyObject())).andThrow(new RuntimeException());
-        replay(producer);
-
-        // since topic name is empty, this method should be a NOOP.
-        // if it attempts to log to the DLQ via the producer, the send mock will throw a RuntimeException.
+        // since topic name is empty, this method should be a NOOP and producer.send() should
+        // not be called.
         deadLetterQueueReporter.report(context);
+        verify(producer, never()).send(any(), any());
```

Comment:
This will lose coverage if we ever start using a different variant of `Producer::send` and forget to update the method signature here (which may happen since the test won't start failing in that case as well).

Probably safer to ensure that we have no interactions whatsoever with it:
```suggestion
                verifyNoInteractions(producer);
```